src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  right: 0;
+  left: 0;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  opacity: 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo {
+  display: block;
+  position: absolute;
+  left: 50px;
+  top: 10px;
+  background-image: url(./images/logo.png);
+  background-repeat: no-repeat;
+}
+
+.nav__list {
+  position: relative;
+  left: 377px;
```

Comment:
What is 377px? Will be better to use display: flex for header and set position of navigation using justify-content property